src/main/java/no/nav/veilarbveileder/service/AuthService.kt

```diff
@@ -62,8 +58,12 @@ class AuthService(
         }
     }
 
-    fun harModiaAdminRolle(ident: NavIdent?): Boolean {
-        return ldapClient.veilederHarRolle(ident, ROLLE_MODIA_ADMIN)
+    @Cacheable(CacheConfig.VEILEDER_ROLLE_CACHE_NAME)
```

Comment:
This cache is valid for 12 hours....what you think is that too much or its ok?

Reply:
hmm, generally i don't think it is a role that is "switched" a lot - guessing you need to be in a specific possition to have the modia_admin role. Dont know if we have had any issues with if previous? 
However i don't mind setting a shorter timespan :P What do you think it should be?